csatar-theme/pages/szovetseg.htm

```diff
@@ -0,0 +1,16 @@
+url = "/szovetseg/:id/:action?"
+layout = "default"
+title = "Szövetség"
+
+[basicForm]
+formId = 1
+recordKeyParam = "id"
+recordActionParam = "action"
+readOnly = 1
+createRecordKeyword = "create"
+actionUpdateKeyword = "update"
+actionDeleteKeyword = "delete"
```

Comment:
```suggestion
createRecordKeyword = "letrehozas"
actionUpdateKeyword = "modositas"
actionDeleteKeyword = "torles"
```
Itt magyarul kellene legyenek a kulcsszavak. Most esik le, hogy én is rosszul csináltam a https://github.com/csatar/csatar-october/blob/main/csatar-theme/pages/tag.htm oldalon. Kijavítanád ott is légy szíves?